src/platform/esp32/architecture.h

```diff
@@ -53,6 +53,8 @@
 #ifdef PIN_BUTTON_TOUCH
 #define BUTTON_PIN_TOUCH PIN_BUTTON_TOUCH
 #endif
+
+#define BUTTON_PIN (-1)
 //
```

Comment:
This woud belong in configuration.h and urgently with a check if it is not already set.

Reply:
Moved, rebased commit on top of master HEAD and force-pushed my branch.